src/poliastro/coordinates.py

```diff
@@ -0,0 +1,169 @@
+"""Functions related to coordinate systems and transformations.
+
+This module complements :py:mod:`astropy.coordinates`.
+
+"""
+
+import math
+
+import astropy.time as time
+import astropy.coordinates as coordinates
+import astropy.units as u
+
+from poliastro.util import transform
+
+
+T = 0
+d = 0
+J2000 = time.Time('J2000', scale='tdb')
```

Comment:
J2000 is already defined elsewhere:

https://github.com/poliastro/poliastro/blob/f6676ef/src/poliastro/twobody/orbit.py#L18

I would rather move it to `constants.py`. It doesn't need to be an Astropy `Constant`, but it clearly does not belong there.